File: activity/20200805/junhaolin/26.Remove_Duplicates_from_Sorted_Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        for (auto it=nums.begin(); it!=nums.end(); it++) { 
            while (it+1!=nums.end() && *it==*(it+1))
                nums.erase(it+1);
        }
        return nums.size();
    }
};

File: activity/20200805/ninezero/26_remove_duplicates_from_sorted_array.go
func removeDuplicates(nums []int) int {
	p, q := 0, 1
	for q < len(nums) {
		if nums[p] != nums[q] {
			if p + 1 < q {
				nums[p+1] = nums[q]
			}
			p++
		}
		q++
	}
	return p+1
}
